server/api/order/order.controller.js

```diff
@@ -22,10 +23,28 @@ exports.show = function(req, res) {
 
 // Creates a new order in the DB.
 exports.create = function(req, res) {
-  Order.create(req.body, function(err, order) {
-    if(err) { return handleError(res, err); }
-    return res.json(201, order);
-  });
+  console.log(req.body.total);
+  var charge = stripe.charges.create({
+      amount: req.body.total,
+      currency: 'usd',
+      card: req.body.billing.stripeToken,
+      description: "email@email.com"
+    }, function(err,charge) {
+          if(err && err.type === 'StripeCardError') {
```

Comment:
i think the error message *would* be different, so there's no need for two different error handlers.